src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/WebWorker.cs

```diff
@@ -38,13 +38,13 @@ public static Task<T> RunAsync<T>(Func<Task<T>> body, CancellationToken cancella
                     // the continuation is executed by setTimeout() callback of the thread.
                     res.ContinueWith(t =>
                     {
-                        PostWhenDone(parentContext, tcs, res);
+                        SendWhenDone(parentContext, tcs, res);
```

Comment:
Why are we changing these from post to send?

Reply:
It became bit more stable after this change, but I'm sure if we should go one or the other way. This is more conservative.

In case of JSException, it would have chance to print the JS stack trace on the receiving side before the webWorker is detached.

Reply:
Note that we will not include this class in public API of Net8, so we could continue improving it.

Reply:
My concern is that Send can deadlock in ways that Post cannot, as I understand it. So I'm wary of using it unless it's necessary.

Reply:
this PR only renames it. I agree with the concern. 

Steve gave similar feedback on the Blazor PR https://github.com/dotnet/aspnetcore/pull/48991#discussion_r1263731418
They have interesting solution for error handling.